utils/deps.go

```diff
@@ -1,14 +1,16 @@
 package utils
 
 import (
+	"plugin"
+
 	"github.com/Zettablock/zsource/configs"
 	"golang.org/x/exp/slog"
 	"gorm.io/gorm"
-	"plugin"
 )
 
 type Deps struct {
 	SourceDB            *gorm.DB
+	SourceDBSchema      string
```

Comment:
SourceDBSchema already in config

we do not need it

in fact, i plan to remove `DestinationDBSchema`